com.github.gkarsay.parlatype.json

```diff
@@ -0,0 +1,58 @@
+{
+    "app-id": "com.github.gkarsay.parlatype",
```

Comment:
I know this matches upstream but wouldn't `io.github` be a better prefix since you actually use that domain?

Reply:
`com.github` is where the code is hosted, `io.github` is where the program is described. As far as I know there is no rule which one should be used. I think the important thing is that the app-id is **unique** and independent of the distributor. That's why I would prefer to use the upstream id.